.coveragerc

```diff
@@ -9,7 +9,187 @@ omit =
     homeassistant/helpers/typing.py
     homeassistant/helpers/signal.py
 
-    # omit pieces of code that rely on external devices being present
```

Comment:
Seems too many files were changed in this PR.

Reply:
what does it mean? I only change files contained on ebus component folder

Reply:
The diff for this file is showing a lot of different changes unrelated to your files. Please remove those changes.

Reply:
I didn't change anything that is not concerning my folder. I don't understand why this is marked as changed

Reply:
The structure of the file has changed. See: https://github.com/home-assistant/home-assistant/blob/dev/.coveragerc

I'm guessing something happened when you pulled in `dev` at some point.

To continue forward, you can put back the version from `dev` and then add your single line.

Reply:
I just spotchecked all of the things it's still seeing as different here and it's looking good to me.